packages/react-conformance/src/types.ts

```diff
@@ -1,9 +1,12 @@
-export interface IsConformantOptions {
+import { ComponentDoc } from 'react-docgen-typescript';
+export interface TestingOptions {
```

Comment:
Generally nice to have a blank line between imports and contents.
```suggestion

export interface TestingOptions {
```